wallet/internal/txsizes/size.go

```diff
@@ -125,3 +126,12 @@ func EstimateSerializeSize(scriptSizers []ScriptSizer, txOuts []*wire.TxOut, add
 func EstimateInputSize(scriptSize int) int {
 	return 32 + 4 + 1 + 8 + 4 + 4 + wire.VarIntSerializeSize(uint64(scriptSize)) + scriptSize + 4
 }
+
+// EstimateOutputSize returns the worst case serialize size estimate for a tx output
+// 	- 8 bytes amount
```

Comment:
the indent here is different then in the function above, which uses two spaces after the beginning of the comment